tests/scripts/single_card/run_demos_single_card_n300_tests.sh

```diff
@@ -1,5 +1,28 @@
 #/bin/bash
 
+run_n300_falcon7b_tests() {
+  # Record the start time
+  fail=0
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_falcon7b_tests"
+
+  # Perf verification for 128/1024/2048 seq lens
+  pytest -n auto --disable-warnings -q -s --input-method=json --input-path='models/demos/falcon7b/demo/input_data.json' models/demos/wormhole/falcon7b/demo_wormhole.py::test_demo[user_input0-perf_mode_128_stochastic_verify] ; fail+=$?
+  pytest -n auto --disable-warnings -q -s --input-method=json --input-path='models/demos/falcon7b/demo/input_data.json' models/demos/wormhole/falcon7b/demo_wormhole.py::test_demo[user_input0-perf_mode_1024_stochastic_verify] ; fail+=$?
```

Comment:
Hey so per #7586 , do you mind restructuring your files + tests so that you can do `pytest -n auto --disable-warnings -q -s --input-method=json --input-path='models/demos/falcon7b/demo/input_data.json' models/demos/wormhole/falcon7b/demo_wormhole_ci.py` or something like that? We don't want to target with `::[...]` in pytest. It's a to-do item to convert all remaining ones to that eventually.

Reply:
As we discussed I have created and assigned myself #10307 to take care of this as soon as I can. Need the changes in this PR for some README updates gating the wormhole release.